app/main.py

```diff
@@ -1,6 +1,49 @@
+import os
+
+import requests
+from requests import RequestException, Response
+
+URL = "http://api.weatherapi.com/v1/current.json"
+API_KEY = os.environ.get("API_KEY")
+FILTERING = "Paris"
+
+
+def get_response() -> Response:
+    try:
+        response = requests.get(URL + f"?key={API_KEY}&q={FILTERING}")
+    except ConnectionError:
+        raise ConnectionError(
+            "There was a problem connecting to the resource."
+        )
+    except RequestException:
+        raise RequestException("An error occurred while making the request.")
```

Comment:
You will never receive an error :) Fix it. You need to raise an error. But how?

Reply:
Done